src/mds/MDSMap.cc

```diff
@@ -141,6 +141,10 @@ void MDSMap::dump(Formatter *f) const
 {
   f->dump_int("epoch", epoch);
   f->dump_unsigned("flags", flags);
+  f->dump_bool("not_joinable", not_joinable());
+  f->dump_bool("allow_snaps", allows_snaps());
+  f->dump_bool("allow_multimds_snaps", allows_multimds_snaps());
+  f->dump_bool("allow_standby_replay", allows_standby_replay());
```

Comment:
@batrick We could check which flags are present by iterating the flagmap and dump only those, alternatively

Reply:
This is fine but please put it in an object.